build/packages-template/bbb-html5/before-install.sh

```diff
@@ -1,28 +1 @@
 #!/bin/bash -e
-
-addGroup meteor ""
-addUser meteor "" meteor /usr/share/meteor "meteor user-daemon" "/usr/sbin/nologin"
-
-case "$1" in
-  install|upgrade|1|2)
-
-    if [ -f /usr/share/meteor/bundle/programs/server/node_modules ]; then
-      rm -r /usr/share/meteor/bundle/programs/server/node_modules
-    fi
-
-    # Remove remnants from old architecture prior to BBB 3.0.x-alpha.6
-    if [ -f /usr/lib/systemd/system/bbb-html5-backend@.service ]; then
-      rm /usr/lib/systemd/system/bbb-html5-backend@.service
-    fi
-    if [ -f /usr/lib/systemd/system/bbb-html5-frontend@.service ]; then
```

Comment:
Please keep these cleanup procedures - they were put to handle upgrades from earlier 3.0 BBB installations. We can remove them safely in the future (3.1+)